mmv1/products/gkebackup/api.yaml

```diff
@@ -0,0 +1,238 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: GKEBackup
+display_name: Backup for GKE
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://gkebackup.googleapis.com/v1/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Backup for GKE API
+    url: https://console.cloud.google.com/apis/library/gkebackup.googleapis.com
+objects:
+  - !ruby/object:Api::Resource
+    min_version: beta
```

Comment:
today, enabling backup for gke is done as an addon feature in gke cluster config, and the add on itself is still in beta. I thought there would be a test failure in the GA tests since the add on can't be enabled in GA. 

I was going to wait to merge this CL and then remove the beta restriction on the gke side and backup for gke side in a follow up CL